docker/config/nginx-prod

```diff
@@ -0,0 +1,37 @@
+FROM nginx
+
+# Add our own website configuration.
+ADD ./docker/config/nginx.conf /etc/nginx/conf.d/default.conf
+
+# Install OpenSSL
+RUN apt-get update && apt-get install openssl -y
+
+# Create SSL certificate directory (where we will put our self-signed certificate and key)
+RUN mkdir /etc/nginx/ssl
+
+# Generate a self-signed certificate and key
+RUN openssl req -new -x509 -days 365 -nodes -newkey rsa:2048 \
```

Comment:
Please note that you are generating a self-signed certificate for the production contianer.